static/css/main.css

```diff
@@ -31,78 +31,74 @@ figure {
   margin: 0;
 }
 
-a {
-  color: #279e6a;
-  color: #279e6aee;
+
+.btn {
+  display: block;
+  color: inherit;
   text-decoration: none;
-  transition: color .3s;
+  background-color: #f9f9f9;
+  border: 1px solid #ccc;
+  border-radius: 5px;
+  padding: 3px 15px;
+  transition: all .3s ease;
 }
-a:hover {
-  color: #00bf6b;
-  color: #00bf6bee;
+.btn:hover {
+  color: #000;
+  background-color: #eee;
 }
 
 /* Header */
 body > header {
```

Comment:
This was applied: https://github.com/neomad-team/neomad.org/blob/master/static/css/main.css#L46